pandas/_testing/__init__.py

```diff
@@ -279,6 +281,28 @@ def box_expected(expected, box_cls, transpose=True):
     return expected
 
 
+def wrap_value(value, cls, transpose=False):
```

Comment:
The motivation for this was to simplify writing tests that should produce the same output for both scalar and container types, e.g. [here](https://github.com/pandas-dev/pandas/pull/46854/files#diff-fbcbe1c510363cf465fa8eb7f5cf8fc3f6bfcc9e1fab7b93f1b69521de44d194R2209).

Reply:
are there any other thematically similar functions that could be grouped together in a pandas._testing.foo instead of `__init__`?  we've kind of stagnated, but in principle there's a goal to get stuff out of this file